rest-api/src/test/java/gov/cms/qpp/conversion/api/acceptance/ValidationApiAcceptance.java

```diff
@@ -0,0 +1,65 @@
+package gov.cms.qpp.conversion.api.acceptance;
+
+import gov.cms.qpp.conversion.model.error.AllErrors;
+import gov.cms.qpp.conversion.model.error.Detail;
+import gov.cms.qpp.conversion.xml.XmlException;
+import gov.cms.qpp.conversion.xml.XmlUtils;
+import gov.cms.qpp.test.annotations.AcceptanceTest;
+import io.restassured.response.Response;
+import org.jdom2.Attribute;
+import org.jdom2.DataConversionException;
+import org.jdom2.filter.Filter;
+import org.jdom2.filter.Filters;
+import org.jdom2.xpath.XPathExpression;
+import org.jdom2.xpath.XPathFactory;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.fail;
+import static com.google.common.truth.Truth.assertThat;
+
+
+@ExtendWith(RestExtension.class)
+class ValidationApiAcceptance {
+	private static final XPathFactory XPF = XPathFactory.instance();
+	private static final Path PATH = Paths.get("../sample-files/CPCPlus_Validation_API_Errors.xml");
+	private static final int CANNED_VALUE = 1000;
+
+	@AcceptanceTest
+	void testUnprocessedFiles() {
+		Response response = given()
+			.multiPart("file", PATH.toFile())
+			.when()
+			.post("/");
+
+		AllErrors blah = response.getBody().as(AllErrors.class);
+		blah.getErrors().get(0).getDetails()
```

Comment:
why not blah.getErrors().stream().map(Error::getDetails).forEach(this::verifyDetail)